config/locales/en.yml

```diff
@@ -1422,6 +1422,10 @@ en:
           only_one_trial: "Only one trial token can be active. Please delete the previous trial token before adding another."
           unreadable: "can't be read. Are you sure it is a support token?"
           already_added: "This token has already been added."
+        enumeration:
+          attributes:
+            active:
+              active_required: "needs the 'Active' property to be true."
```

Comment:
Since the error is added on the `is_default` attribute, even if the validation message would be visible, the translation would still be missing, because the translation string is added on the `:active` attribute.
I also think we might not need this.